app/src/main/res/values-fr/strings.xml

```diff
@@ -225,9 +225,9 @@
     <string name="configLabel_general_calculator_sun">Source de données pour le soleil</string>         <!-- preference label -->
     <string name="configLabel_general_calculator_moon">Source de données pour la lune</string>         <!-- preference label -->
 
-    <string name="configLabel_general_timeFormatMode">Forfait d'eheure :</string>         <!-- spinner label -->  <!-- TODO -->
+    <string name="configLabel_general_timeFormatMode">Forfait d\'eheure :</string>         <!-- spinner label -->  <!-- TODO -->
```

Comment:
`'` is a special character. It needs to be escaped with a `\`

Reply:
Thanks and sorry but there are a big typo error we need to correct, translation should be: "Format des heures"

Reply:
fixed. I picked that line at random. Lucky review I guess.